lib/find-view.js

```diff
@@ -593,9 +594,27 @@ class FindView {
       if (this.model.getFindOptions().useRegex) {
         findPattern = Util.escapeRegex(findPattern);
       }
+
       if (findPattern) {
         this.findEditor.setText(findPattern);
-        this.search();
```

Comment:
Looks like this no longer activates the search immediately, is that the desired behavior?

Reply:
Ah you're right, I thought this wasn't doing anything useful, but it turns out it does have a purpose if live search is disabled. I'll push a new commit fixing that.